src/controllers/update/update.review.overseas.entity.information.controller.ts

```diff
@@ -32,3 +34,21 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     next(errors);
   }
 };
+
+const getBackLinkUrl = (appData: ApplicationData) => {
+  let backLinkUrl;
+
+  const agentDueDiligence = appData[DueDiligenceKey] && Object.keys(appData[DueDiligenceKey]).length > 0;
+  const overseasEntityDueDiligence = appData[OverseasEntityDueDiligenceKey] && Object.keys(appData[OverseasEntityDueDiligenceKey]).length > 0;
```

Comment:
you could move line 42 into the else part of first if else { const ...; if (overseasEntityDueDiligence) {...} } else { ..} but that's just an optimization.